draft-wang-ppm-differential-privacy.md

```diff
@@ -319,12 +325,114 @@ or do we just mention the various types briefly and detail in a separate section
 ### Randomized Response
 
 
-# Apply Differential Privacy Policy In a Secure Agggregator Settting {#dp policy}
+# Apply Differential Privacy Policy In a Secure Aggregator Setting {#dp policy}
 
 > OPEN ISSUE: or call it dp strategy?
 
-> TODO: Junye to fill in, expand on aggregator DP, aggregator DP amplified by
-local DP, includes contents in "Execution of an Aggregation Protocol with DP"
+An Aggregator MUST enforce the privacy guarantee of its aggregate share before
+it outputs its aggregate share.
+
+Therefore, we will define a class `DpPolicy` used by an Aggregator to enforce
+the privacy guarantee. A concrete implementation of `DpPolicy` should provide
+the necessary parameters that quantify the DP guarantee to the initializer, and
+provide the implementation in `guarantee_privacy` method to accomplish such
+guarantee.
+
+~~~
+class DpPolicy:
+    AggShare
+
+    def guarantee_privacy(self,
+                          agg_share: AggShare,
+                          num_reports: int,
+                          min_batch_size: int) -> AggShare:
+        pass
+~~~
+
+An Aggregator can enforce the DP policy in one of the following ways:
+
+## Pure Aggregator DP {#dp-policy-pure-aggregator}
+
+If the target privacy guarantee is provided only by Aggregators, each Aggregator
+can use a concrete implementation of `DpMechanism` to enforce Aggregator-DP.
+
+## Composition of Aggregator DP with Local DP {#dp-policy-composition}
```

Comment:
How about "Client-DP" rather than "Local-DP"?

We started saying "Aggregator-DP" instead of "Central-DP" due to the taxonomy in https://petsymposium.org/popets/2023/popets-2023-0086.pdf. We would say "Client-DP" rather than "Local-DP" to be consistent. 

I guess I don't have a strong opinion about which set of terms to use, so long as we're consistent.